charts/external-dns/templates/crd.yaml

```diff
@@ -0,0 +1,103 @@
+{{- if .Values.deployCrd }}
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
```

Comment:
note to myself: I am missing `api-approved.kubernetes.io: "https://github.com/kubernetes-sigs/external-dns/pull/2007"` key-value pair here. This is not generated by `controller-gen` and can't be added as a Kubebuilder annotation afaik.